### PR TITLE
Aleph port

### PR DESCRIPTION
**Pull request by jedwards4b**
Port to aleph plus a bug fix in chunking.py - variables calendar and units are not defined if size<=1

### CODE REVIEW

File: Machines/aleph_modules [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# >>> conda initialize >>>
+# !! Contents within this block are managed by 'conda init' !!
+__conda_setup="$('/home/jedwards/anaconda2/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
```
Comment (bertinia) [inline Machines/aleph_modules]: anyway to make this path non-user specific? perhaps. an env var instead?

Comment (jedwards4b) [reply Machines/aleph_modules]: All the users on this system are using my conda install.

File: timeseries/template.timeseries [hunk 2]
```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env python
-{{ batchdirectives }}
+#PBS -N timeseries.{{ case }}
+#PBS  -r n 
+#PBS  -j oe 
+#PBS  -V 
+#PBS -l select=10:ncpus=40:mpiprocs=10
```
Comment (bertinia) [inline timeseries/template.timeseries]: @jedwards4b - This should stay as a template variable {{ batchdirectives }} for
portability. Is there are reason that you made it specific to PBS?

File: timeseries/template.timeseries [hunk 3]
```diff
@@ -38,14 +64,17 @@ def _main_func(description):
             "unit_testing" : False
             }
         executable, mpi_arg_list = env_mach_specific.get_mpirun(case, mpi_attribs, job, overrides=overrides)
-        cmd = executable+" "+" ".join(mpi_arg_list) + " " + run_exe
+#        cmd = executable+" "+" ".join(mpi_arg_list) + " " + run_exe
+        cmd = "/home/jedwards/anaconda2/envs/ppenv/bin/mpirun " + run_exe
```
Comment (bertinia) [inline timeseries/template.timeseries]: again, this shouldn't be a machine / user specific path

**Pull request merged**